src/agri_ai/line_bot/app.py

```
@@ -0,0 +1,321 @@
+        
+        # メッセージハンドラー統計
+        processing_stats = message_handler.get_processing_stats()
+        
+        # データベース統計
+        db_stats = {}
+        if hasattr(optimized_db, 'get_database_stats'):
+            db_stats = await optimized_db.get_database_stats()
+        
+        return {
+            "status": "ok",
+            "timestamp": datetime.now().isoformat(),
```

Ruff (0.11.9)

Undefined name `datetime`

(F821)